### DISCUSSION TITLE
Quadruped Robot with a Manipulator (Project 29)

### DISCUSSION THREAD

**Discussion by robertogl** [mathworks/MATLAB-Simulink-Challenge-Project-Hub#6] [Collaborate]
Contribute to the discussion by asking and/or answering questions, commenting, or sharing your ideas for solutions to [project #29](https://github.com/mathworks/MathWorks-Excellence-in-Innovation/tree/main/projects/Quadruped%20Robot%20with%20a%20Manipulator)

---
*:robot: An AI bot may reply to your comments with suggestions and guidance. It will not solve the project for you. Your comments may be processed by a third-party LLM. To direct a question to a specific person, tag them with @username.*

**Comment by prathamb** [comment]
Hello , I am Prathamesh from the Indian Institute of Technology Roorkee. 
I have recently started to learn robotics.
Looking forward to collaborate and learn new things

Thank you

**Comment by sriram-2502** [comment]
Hi, I'm Sriram. I am a Ph.D. candidate at Clemson University working on robotics and data-driven control. I am interested in developing Matlab and Simulink based solutions for designing controllers for quadruped locomotion with a manipulator. Excited to work with other people with similar interests and develop new algorithms for this project.

**Comment by SheriffEllis** [comment]
Hello, my name is Sherif. I am a student pursuing a masters in Mechatronic Engineering at The University of Sheffield. We have been working on adapting the model to perform in a 3D physics environment. One issue we have encountered recently is that we cannot find where the model parameters are initialised from. They appear to be initialised at compile time from within the model but I cannot find a file or block to edit that would allow me to edit these workspace parameters.

**Reply by smiller01985** [reply]
Hi Sherif.  The model loads parameters in the InitFcn
![image](https://user-images.githubusercontent.com/56474268/219150461-d4bdc174-ddb5-4516-ade5-e29177850f46.png)
--Steve

**Comment by CarloCarlo25** [comment]
"Hi, I am a student at University of the Philippines Los Baños and interested in working and collaborating on this project"

**Comment by nidhibaratam** [comment]
Hi, I am Baratam NidhiShri, a student at LNM Institute of Information Technology  and interested in working and collaborating on this project.I am undergraduate sophomore studying electronics and communication engineering . This is my first project .I hope to learn something new through this project.

**Comment by BeNavon** [comment]
Which papers did you base the [background material](https://github.com/mathworks/MATLAB-Simulink-Challenge-Project-Hub/tree/main/projects/Quadruped%20Robot%20with%20a%20Manipulator#background-material) on? 
Particularly regarding the dynamics equations and control loop.

**Comment by katsrebe** [comment]
Hey, I’m Berk Esat, a recent graduate of Istanbul University-Cerrahpaşa with a degree in Mechanical Engineering. I’m really excited about this challenge and looking forward to learning a lot.
